templates/base.html

```diff
@@ -49,24 +50,30 @@
                     <div class="col-lg-4 col-lg-offset-3 col-md-5 col-md-offset-2 col-sm-7 col-sm-offset-1 col-xs-8">
                             <p>
                                 {% if request.session.cobrand_active %}
-                                    <img alt="{{ request.session.cobrand_company_name }} Logo" src="{{ MEDIA_URL }}cobrand/{{ request.session.cobrand_company_logo }}"/>
+                                    {% cache 86400 cobrand_header_logo request.session.cobrand_company_logo %}
```

Comment:
@waseem-omar do we even need this now that we're caching the underlying data? if we don't cache this, we don't have a roundtrip to redis, and we just do string generation. We are already pulling the data from redis.